src/features/import-projects/helpers.ts

```
@@ -45,18 +45,37 @@ export const updateOrCreateProject = async (
-      await dataSource
-        .createQueryBuilder()
-        .delete()
-        .from(Project)
-        .where("id = :id", { id })
-        .execute();
-
-      console.log(
-        `[${new Date().toISOString()}] - INFO: Project Deleted. Project ID: ${id}`
-      );
+    // If project is imported, delete it from the database
+    if (existingProject && existingProject.imported) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)